### DISCUSSION TITLE
Did Sept 3rd's tqdm-4.64.1 miss picking up Sept 2nd's commit?

### DISCUSSION THREAD

**Discussion by aifartist** [tqdm/tqdm#1427] [Q&A] [answered]
Sept 2nd's commit of https://github.com/tqdm/tqdm/commit/d2f2366f7b8e118ed7527905776da57883e66084
fixes https://github.com/tqdm/tqdm/issues/1288
but missed the release and even today it still isn't in master.

**Comment by casperdcl** [comment, marked as answer]
The pull request is still open (#1332) - please do continue discussion there :)
